### ISSUE TITLE
Papers on which the Overfeat training code of TensorBox is based

### ISSUE THREAD

**Issue by rupakl** [Russell91/TensorBox#276]
I want to know if the overfeat training code based on these two papers.
1.) [https://arxiv.org/abs/1312.6229](https://arxiv.org/abs/1312.6229)
2[https://arxiv.org/pdf/1612.07695.pdf](https://arxiv.org/pdf/1612.07695.pdf) for rezoom portion.
And the loss function which is used.

**Comment by RomanSteinberg** [comment]
Right. I'm not sure what loss function do you mean. You can look at TB code to find out that loss is a sum of three losses.

**Comment by rupakl** [comment]
Thanks. I have figured out the loss portion.
